### ISSUE TITLE
Having an issue building for XCode 10 on macOS 10.14

### ISSUE THREAD

**Issue by heyztb** [XVimProject/XVim2#178]
Hello,

As the title suggests, I am having an issue building for XCode 10 on macOS Mojave. I'm using the `develop` branch, which I understand is compatible with Xcode 10, and from what I've seen in other issues here other users have gotten this working. 

Here is the error I'm seeing

    ** BUILD FAILED **

    The following build commands failed:
            PBXCp /Users/zach/XVim2/build/Release/XVim2.xcplugin /~/Library/Application\ 
    Support/Developer/Shared/Xcode/Plug-ins/XVim2.xcplugin

Basically, it looks like PBXCp is being supplied an incorrect path. It should be `~/Library/Application\ Support/Developer/Shared/Xcode/Plug-ins/XVim2.xcplugin` instead. 

I have XVim2 installed just fine on Xcode 9.4 on the same system.

**Comment by pebble8888** [comment]
7c8e72c1 fixed in `master` and `develop` branch.
